Shared/en-US.lproj/Localizable.strings

```diff
@@ -1033,6 +1033,12 @@
 /* When making a new search from the awesome bar, suggestions appear to the user as they write new letters in their search. Different types of suggestions can appear. This string will be used as a header to separate Firefox suggestions from normal suggestions. */
 "Search.SuggestSectionTitle.v102" = "Firefox Suggest";
 
+/* When making a new search from the awesome bar, suggestions appear to the user as they write new letters in their search. Different types of suggestions can appear. This string will be used to form the title for a Firefox suggestion, using the completed search keyword (1$) and suggested page title (2$). */
```

Comment:
How was those added? The strings in our Localizable.strings are normally added with an automated script ([example](https://github.com/mozilla-mobile/firefox-ios/pull/16236)) when they are imported from the l10n repo. If you add new strings in the `Strings.swift` file, they will be automatically added to all the translated locales automatically

Reply:
Oh, perfect, thank you! 🙏🏼 I added this one by hand, because I didn't know they were auto-generated—I'll revert this change.